build.savant

```diff
@@ -23,13 +23,13 @@ javaHTTPVersion = "0.3.6"
 jsonPatchVersion = "1.13.0"
 guavaVersion = "32.1.2-jre"
 guiceVersion = "6.0.0"
-jacksonVersion = "2.15.2"
+jacksonVersion = "2.15.4"
 jsoupVersion = "1.16.1"
 logbackVersion = "1.4.14"
-slf4jVersion = "2.0.13"
+slf4jVersion = "2.0.17"
 testngVersion = "7.8.0"
 
-project(group: "org.primeframework", name: "prime-mvc", version: "4.29.3", licenses: ["ApacheV2_0"]) {
+project(group: "org.primeframework", name: "prime-mvc", version: "4.29.4", licenses: ["ApacheV2_0"]) {
```

Comment:
@robotdan 4.30.0 is getting released as we speak, so this probably needs to be 4.30.1